examples/test_flow_dir/test_flow_dir.c

```diff
@@ -155,19 +155,27 @@ static int
 packet_handler(struct rte_mbuf *pkt, struct onvm_pkt_meta *meta,
                __attribute__((unused)) struct onvm_nf_local_ctx *nf_local_ctx) {
         static uint32_t counter = 0;
-        struct onvm_flow_entry *flow_entry = NULL;
+        struct onvm_flow_entry *flow_entry;
```

Comment:
Initializing this to NULL resulted in a seg fault when creating the chain flow_entry->sc.

Reply:
It seems unlikely that removing the NULL is actually fixing the problem - it may just be masking the error.  It is better to initialize it to NULL and then figure out why it is being accessed before being initialized properly.